source/grid/grid_generator.cc

```diff
@@ -3784,7 +3784,8 @@ namespace GridGenerator
   void
   merge_triangulations(const Triangulation<dim, spacedim> &triangulation_1,
                        const Triangulation<dim, spacedim> &triangulation_2,
-                       Triangulation<dim, spacedim> &      result)
+                       Triangulation<dim, spacedim> &      result,
+                       const double duplicated_vertex_tolerance)
```

Comment:
align it if you can and clang-format doesn't undo it

Reply:
clang-format always puts it this way :(